src/serialmonitor/usbDetector.ts

```diff
@@ -106,6 +117,20 @@ export class UsbDetector {
         }
     }
 
+    public pauseListening() {
+        if (this._usbDetector) {

```

Comment:
In fact, it is possible but the startListening() method needs to be modified. The current startListening() just sets the add event listener and does not explicitly start listening. It works previously just because the detector is default to be listening after initialization. startListening() cannot switch back the status of detector correctly yet.

My concern is that it may be unnecessary to set the add event listener again and again.